frontend/src/components/App/runCommand.ts

```diff
@@ -41,9 +48,24 @@
  * ```
  */
 export function runCommand(
-  command: 'minikube' | 'az',
+  command: 'minikube' | 'az' | 'scriptjs',
   args: string[],
-  options: {}
+  options: {},
+  permissionSecrets?: Record<string, number>,
```

Comment:
consider adding these three args as an option

Reply:
We already have an options object, and the user is given a runCommand wrapper to use which only uses the command, args, and options params as outlined in the examples.

So the extra arguments will stay as is.